FAQ.md

```diff
@@ -0,0 +1,44 @@
+# Frequently asked questions
+
+## Getting started
+
+First you need to clone the repository. You can do this with
+
+```bash
+git clone https://github.com/WhatsApp/proxy.git
+```
+
+## Common issues
+
+### (1) The container won't build on Windows with `set_public_ip_and_start.sh: Not found`
```

Comment:
Doing these as headers, as you've done, should result in a nice [auto-generated table of contents](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/) 👍 .

If we end up with a lot of questions, we may want to manually maintain a ToC at the top to make it easier for people to find what they're after.